src/AI/SpenderAI.java

```diff
@@ -187,16 +187,7 @@ else if( unitQueue.isEmpty() )
             boolean validTarget = false;
             ArrayList<XYCoord> validTargets = new ArrayList<>();
 
-            boolean canCapture = false;
-            for( ActionType i : unit.model.possibleActions )
-            {
-              if( i == ActionType.CAPTURE )
-              {
-                canCapture = true;
-                break;
-              }
-            }
-            if( canCapture )
+            if( unit.model.possibleActions.contains(ActionType.CAPTURE) )
```

Comment:
Hah. In the work I'm doing right now, I added a `canPerformActionType(ActionType)` function to `UnitModel`.